plrust/src/tests.rs

```diff
@@ -264,7 +262,7 @@ mod tests {
                 IMMUTABLE STRICT
                 LANGUAGE PLRUST AS
             $$
-                Some(::pgx::iter::SetOfIterator::new(names.into_iter().map(|maybe| maybe.map(|name| name.to_string() + " was booped!"))))
+                Ok(::pgx::iter::SetOfIterator::new(names.into_iter().map(|maybe| maybe.map(|name| name.to_string() + " was booped!"))))
```

Comment:
Needs to double-wrap.
```suggestion
                Ok(Some(::pgx::iter::SetOfIterator::new(names.into_iter().map(|maybe| maybe.map(|name| name.to_string() + " was booped!")))))
```

Reply:
Yeah, this one just popped up in CI.  :face_palm:

Reply:
This kinda thing makes it look like I didn't run the tests locally first.  EVEN THO I DID I SWEAR

Reply:
This is all been fixed up in pgx 0.7.0-beta.1